spec/lib/stellar/client/assets_spec.rb

```diff
@@ -0,0 +1,57 @@
+require "spec_helper"
+
+describe Stellar::Client do
+  subject(:client) { Stellar::Client.default_testnet }
+
+  describe "#assets" do
+    let(:asset_issuer) { Stellar::Account.from_address('GD4SAUKGB6GE2Q25H2CZMZ3BSP5CVYIY2LQYJDCFNNICR473AVL7IYH5') }
```

Comment:
Will this cause issues in the future when the asset issuer no longer exists because something was done to it (i.e. merged into another account)?

Reply:
@ccjr you could use https://github.com/stellar/ruby-stellar-sdk/blob/master/spec/config.yml.sample (copy it to `spec/config.yml`) to specify an account on the testnet that has lumens. Then, [you can make other accounts in the specs and fund them](https://github.com/stellar/ruby-stellar-sdk/blob/master/spec/lib/stellar/client_spec.rb#L81) using that account. We'll have to do this until we can use friendbot.